src/components/website-management/website-form-components/customization/customization-components/CookieBanner.js

```diff
@@ -0,0 +1,271 @@
+        loadTemplates();
+        laodBanner();
+    }, []);
+
+
+    const regulationValues = templates?.length > 0 ? templates.map(template => ({ id: template.templateId, value: template.templateRegulation }))
+        : [];
+
+    const choosenTemplate = (e) => {
+        const selectedRegulation = e.target.value;
+        let chosenTemp=null;
+        let chosenCont=null;
+
+        for (let i = 0; i < templates.length; i++) {
+            if (templates[i].templateRegulation === selectedRegulation) {
+                chosenCont= templates[i].templateContent;
+                break;
+            }
+        }
+
+        const template = templates.find((t) => t.templateRegulation === e.target.value);
+        setSelectedTemplate(template);
+        setChosenCont(chosenCont);
+    };
+
+    function handleChangeTop(value){
+        if(value==="Top" ){
+            setTop(10);
+            setPositionValue(value);
+        }
+        if(value==="Bottom" ){
+            setTop(280);
+            setPositionValue(value);
+        }
+    }
+
+    function handleChangeHorizontal(value){
+        if(value==="Left"){
+            setLeft(10);
+            setAlignmentValue(value);
+        }
+        if(value==="Center"){
+            setLeft(240);
+            setAlignmentValue(value);
+        }
+        if(value==="Right"){
+            setLeft(475);
+            setAlignmentValue(value);
+        }
+    }
+
+
+    function handleChangeColor(value){
+        if(value==="Yellow" || value==="#F1C40F"){
+            setColor("#F1C40F");
+            setColorValue("Yellow");
+        }
+        if(value==="Dark Gray" || value==="#3f454a"){
+            setColor("#34495E");
+            setColorValue("Dark Gray");
+        }
+        if(value==="Purple" || value==="#9B59B6"){
+            setColor("#9B59B6");
+            setColorValue("Purple");
+        }
+    }
+
+    const handleSnackbarClose = () => {
+        setSnackbarOpen(false);
+        setError(false);
+        setErrorMessage('');
+    };
+
+
+
+    async function handlesave() {
+
+        const websiteID = passedId;
+
+        const bannerSave = {
+            bannerPosition: positionValue,
+            bannerColor: color,
+            bannerAlignment: alignmentValue,
+        };
+
+        try {
+            const response = await axios.put(`http://localhost:8080/api/banner/updateBanner/${websiteID}`, bannerSave);
+            console.log("success", response.data);
+            setSnackbarMessage('Save Completed!');
+            setSnackbarOpen(true);
+        } catch (error) {
+            console.error("Error saving banner:", error)
+            setError(true);
+            setErrorMessage('An error occurred while saving the banner.');
+        }
+
+        console.log("Top value is", alignmentValue);
+        console.log("Left value is", positionValue);
+        console.log("Color value is", color);
+        console.log("Banner value:", JSON.stringify(bannerSave));
```

Comment:
We can use browser debugger to inspect values during development, instead of adding console log statements. Let's discuss further in a meeting.

```suggestion
```

Reply:
fixed